src/wazuh_modules/wm_sca.c

```diff
@@ -2312,29 +2312,38 @@ static int wm_sca_winreg_querykey(HKEY hKey, const char *full_key_name, char *re
                     int size_available;
 
                 case REG_SZ:
-                case REG_EXPAND_SZ:
+                case REG_EXPAND_SZ: {
                     snprintf(var_storage, MAX_VALUE_NAME, "%s", data_buffer);
-                    break;
-                case REG_MULTI_SZ:
+                }
```

Comment:
declare size_data out of the switch statement, to avoid the usage of curly braces.